cmd/wit-bindgen-go/cmd/generate/generate.go

```diff
@@ -140,25 +148,44 @@ func parseFlags(cmd *cli.Command) (*config, error) {
 		cmd.Bool("versioned"),
 		cmd.Bool("force-wit"),
 		path,
+		verbose,
 	}, nil
 }
 
+// Generate and write the specified Go packages w/ provided bindgen config
+//
+// Parameters:
+//   - packages: A slice of pointers to `gen.Package`s to generate
+//   - cfg: A pointer to a `config` struct
+//
+// Returns:
+//   - error: An error if the package writing process fails, or nil if all
+//     packages are written successfully.
 func writeGoPackages(packages []*gen.Package, cfg *config) error {
-	fmt.Fprintf(os.Stderr, "Generated %d package(s)\n", len(packages))
+	var stderrWriter io.Writer
+	if cfg.verbose {
+		stderrWriter = os.Stderr
+	} else {
+		stderrWriter = io.Discard
+	}
+
+	fmt.Fprintf(stderrWriter, "Generated %d package(s)\n", len(packages))
 	for _, pkg := range packages {
 		if !pkg.HasContent() {
-			fmt.Fprintf(os.Stderr, "Skipping empty package: %s\n", pkg.Path)
+			if cfg.verbose {
```

Comment:
This isn't necessary if the writer is io.Discard.

Reply:
Ah sorry yes, I changed it and forgot to remove this! Will fix!